swri_yaml_util/CMakeLists.txt

```diff
@@ -28,10 +28,10 @@ configure_file(version.h.in ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_INCLUDE_DEST
 find_package(Boost REQUIRED)
 
 include_directories(include
-  ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_INCLUDE_DESTINATION}
-  ${catkin_INCLUDE_DIRS}
   ${Boost_INCLUDE_DIRS}
+  ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_INCLUDE_DESTINATION}
   ${YamlCpp_INCLUDE_DIR}
+  ${catkin_INCLUDE_DIRS}
```

Comment:
As a rule, I prefer to include in the order:

1. package-local
2. catkin
3. system

I do this on the principal that a catkin workspace overlay should take precedence over system packages.